src/Errors.ts

```diff
@@ -71,17 +71,25 @@ export class TSTLErrors {
 
     public static UnsupportedFunctionConversion = (node: ts.Node, name?: string) => {
         if (name) {
-            return new TranspileError(`Unsupported conversion from method to function "${name}".`, node);
+            return new TranspileError(`Unsupported conversion from method to function "${name}". `
+                                      + `To fix, wrap the method in an arrow function.`,
+                                      node);
         } else {
-            return new TranspileError(`Unsupported conversion from method to function.`, node);
+            return new TranspileError(`Unsupported conversion from method to function. `
+                                      + `To fix, wrap the method in an arrow function.`,
+                                      node);
         }
     }
 
     public static UnsupportedMethodConversion = (node: ts.Node, name?: string) => {
         if (name) {
-            return new TranspileError(`Unsupported conversion from function to method "${name}".`, node);
+            return new TranspileError(`Unsupported conversion from function to method "${name}". `
+                                      + `To fix, wrap the function in an arrow function.`,
```

Comment:
Should be a very specific arrow function right?

Reply:
The idea is that function expressions can have their context type deduced based on what they are being assigned to, so a simple arrow function should suffice. There's a few edge cases where this doesn't work and 'this' has to be explicitly stated or you will still get this error, though.